Source/Core/InputCommon/ControlReference/FunctionExpression.cpp

```diff
@@ -646,13 +646,13 @@ class NTimesExpression : public FunctionExpression
     if (args.size() == 1 || args.size() == 2)
       return ArgumentsAreValid{};
     else
-      return ExpectedArguments{"input, [times = 1]"};
+      return ExpectedArguments{"input, times = 1"};
   }
 
   ControlState GetValue() const override
   {
     const ControlState input = GetArg(0).GetValue();
-    const ControlState times = GetArg(1).GetValue();
+    const int times = (GetArgCount() >= 2) ? int(GetArg(1).GetValue()) : 1;
```

Comment:
I think in general, when casting to int, all the other functions do something like this:
`const u32 times = GetArgCount() >= 2 ? u32(std::max(GetArg(1).GetValue() + 0.5, 0.0)) : 1;`